caffe2/proto/caffe2.proto

```diff
@@ -15,23 +15,50 @@ package caffe2;
 message TensorProto {
   // The dimensions in the tensor.
   repeated int64 dims = 1;
+  // The strides of the tensor.
+  repeated int64 strides = 12;
```

Comment:
so, strides imply that the underlying storage is shared. Add a  comment on how it's going to be represented in the format? Shared EXTERNAL storage I assume?

Reply:
Should serialized model contain strict shared storage information? How many of the parameters are actually going to be shared post-serialization, or should they be re-created during the initialization step?

Reply:
Will add more comments to make it clear. Yes, strides should only be used with EXTERNAL storage.